l10n_nl_tax_statement_icp/models/l10n_nl_vat_statement.py

```diff
@@ -50,7 +50,6 @@ def _compute_icp_lines(self):
                 icp_values['partner_id'] = partner_id
                 icp_values['statement_id'] = statement.id
                 newline = IcpLine.create(icp_values)
-                statement.icp_line_ids += newline
```

Comment:
The lines are already created with a `statement_id`. Removing this line fixes a mysterious `ValueError: Expected singleton: l10n.nl.vat.statement.icp.line(15, 15)` in my test.